homeassistant/components/tuya/binary_sensor.py

```diff
@@ -46,6 +46,17 @@ class TuyaBinarySensorEntityDescription(BinarySensorEntityDescription):
 # end up being a binary sensor.
 # https://developer.tuya.com/en/docs/iot/standarddescription?id=K9i5ql6waswzq
 BINARY_SENSORS: dict[str, tuple[TuyaBinarySensorEntityDescription, ...]] = {
+    # DeHumidifier
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48r6jke8e
+    "cs": (
+        TuyaBinarySensorEntityDescription(
+            key=DPCode.FAULT,
+            name="Water tank",
+            icon="mdi:water",
+            device_class=BinarySensorDeviceClass.PROBLEM,
+            on_value=8,  # 8 means tank is full
```

Comment:
8... so there are more? Shouldn't this be a sensor with an enum device class instead?

Reply:
The Tuya entity is a Bitmap. The information I got was that 0 is tank normal and 8 tank full. That's why O chose binary and put onvalue=8. I tested and worked correctly. 
Do you think it can go trought this way ?

Reply:
8 is an off value to check in a bitmap?